### PR TITLE
(WIP) Thehollidayin/front end updates

### PR DESCRIPTION
**Pull request by TheHollidayInn**
[//]: # (Note: See http://habitica.wikia.com/wiki/Using_Habitica_Git#Pull_Request for more info)

[//]: # (Put Issue # or URL here, if applicable. This will automatically close the issue if your PR is merged in)
Fixes put_issue_url_here

### Changes
[//]: # (Describe the changes that were made in detail here. Include pictures if necessary)

[//]: # (Put User ID in here - found in Settings -> API)

----
UUID:

### CODE REVIEW

File: website/client/router.js [hunk 1]
```diff
@@ -26,7 +29,15 @@ export default new VueRouter({
       component: ParentPage,
       children: [
         {path: 'tavern', component: Page},
-        {path: 'inbox', component: Page},
+        {
+          path: 'inbox',
+          component: InboxPage,
+        },
+        {
```
Comment (TheHollidayInn) [inline website/client/router.js]: I'm pretty sure this should be a child route, but the child routes seem to keep the html of the parent. Still looking into this.

Comment (paglias) [reply website/client/router.js]: I think, but double check, that the solution here is to have a main `/inbox` route with two (or more) children routes, the first one having `''` (empty path) as its `path` that will be loaded when the main `/inbox` route is loaded.

Something like this:

- Parent route with shared html `/inbox`
    - Main child route with path `''` with the html to be shown only in the parent route but not in the child routes
    - Other child routes

File: website/client/components/Inbox/InboxConversationPage.vue [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    reply: function reply () {
+      // @TODO: This will be default values based on the conversation and current user
+      let messageToSend = {
+        from: 'TheHollidayInn',
+        fromUserId: 3211,
+        to: 'Paglias',
+        message: this.newMessage,
+        date: new Date(),
+      };
+      this.messages.push(messageToSend);
```
Comment (paglias) [inline website/client/components/Inbox/InboxConversationPage.vue:53]: this logic should become an action inside the store

File: website/client/components/Inbox/InboxConversationPage.vue [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+<script>
+export default {
+  data () {
+    // @TODO: Abstract to Store
+    let messages = [
+      {
+        from: 'Paglias',
+        fromUserId: 1234,
+        to: 'TheHollidayInn',
+        message: 'I love the Gang of Four',
```
Comment (paglias) [inline website/client/components/Inbox/InboxConversationPage.vue:32]: this is psychological terrorism 😛

File: website/client/components/Inbox/InboxConversationPage.vue (hunk 2 above)
Comment (TheHollidayInn) [reply website/client/components/Inbox/InboxConversationPage.vue:53]: Yea, added note at the top :D

**Pull request merged**